aim/web/api/runs/views.py

```diff
@@ -513,3 +515,19 @@ async def delete_runs_batch_api(runs_batch: RunsBatchIn):
     return {
         'status': 'OK'
     }
+
+
+@runs_router.post('/archive-batch/', response_model=StructuredRunsArchivedOut)
+async def archive_runs_batch_api(runs_batch: RunsBatchIn, run_in: StructuredRunsArchivedIn,
```

Comment:
can we change this to pass archive flag as a query parameter? so the API will look like:
`POST /api/runs/archive-batch?archive=true|false`
and the body will be 
```
[
<run-hash-1>,
<run-hash-2>,
...
]
```